README.md

```diff
@@ -145,6 +146,16 @@ It will try to connect to the host `172.17.0.1:9000` by default.
 Make sure to add a file mapping in your IDE:
 `./workspace/customer/project` => `/data/customer/project`
 
+# Update from exsisting environment
+
+If you update from a existing environment, follow these steps:
+- dev down
```

Comment:
All commands should be encapsulated with ` to identify them as commands, not just as text.